internal/config/schema.go

```diff
@@ -45,8 +45,10 @@ type Config struct {
 		BaseURL string `validate:"required,url" yaml:"base_url"`            // e.g. "https://api.platform.sh"
 		AuthURL string `validate:"omitempty,url" yaml:"auth_url,omitempty"` // e.g. "https://auth.api.platform.sh"
 
-		UserAgent string `validate:"omitempty" yaml:"user_agent,omitempty"` // a template - see UserAgent method
+		UserAgent string `validate:"omitempty" yaml:"user_agent,omitempty"`       // a template - see UserAgent method
+		SessionID string `validate:"omitempty,ascii" yaml:"session_id,omitempty"` // the ID for the authentication session - defaults to "default"
```

Comment:
I think these changes aren't used

Reply:
It's not used right now, but it's part of the config anyway and needed in case we decide to interface directly with the credentials manager.

I could remove it, but I think it's not an issue.